offchain/danaswap-ui/views/components/noscript.ejs

```diff
@@ -0,0 +1,8 @@
+<noscript>
+  <p lang="en">
+    The DanaSwap website requires JavaScript. Here are the<a
```

Comment:
The whole website doesn't require JS and is static though so this message is misleading. We should make these `<noscript>` messages more specific to the functionality the user isn't seeing.

Reply:
I reckon it does require JS for interactivity

Reply:
Just for the one or two components

Reply:
maybe we don't need this `noscript` for now?